examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<blockquote>Let's have a look at the code generated above and the data printed. </blockquote><p> </p><p>If we look at the generated code, we will see that it is still identical to the one from cell X -- no conditionals are generated. Why? We ...</p><p> </p><blockquote>We notice that the <code>ci</code></blockquote><p> </p><p>drop "the" as you're referring to a specific object by name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>